website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -1,14 +1,13 @@
 ---
-link: Migrating pre-v1.0.0 Projects
-linkTitle: Migrating pre-v1.0.0 Projects
-weight: 6
-description: Instructions for migrating a Ansible-based project built prior to v1.0.0 to use the new Kubebuilder-style layout.
+link: Migrating Projects from pre-v1.0.0 to the latest release
+linkTitle: Migrating from pre-v1.0.0 to latest
+weight: 200
+description: Instructions for migrating an Ansible-based project built prior to `v1.0.0` (`0.19.x+`) to use the Kubebuilder-style layout which is the default layout adopted by SDK since the `1.0.0` release.
```

Comment:
The 0.19.x only applies to Go projects. Really, the Go migration guide should read `Migrating from pre-v0.19.0 to latest`
```suggestion
description: Instructions for migrating an Ansible-based project built prior to `v1.0.0` to use a Kubebuilder-style.
```